tests/admin_changelist/tests.py

```diff
@@ -672,9 +672,16 @@ def test_get_admin_log_templatetag_custom_user(self):
 class SeleniumFirefoxTests(AdminSeleniumWebDriverTestCase):
 
     available_apps = ['admin_changelist'] + AdminSeleniumWebDriverTestCase.available_apps
-    fixtures = ['users.json']
     webdriver_class = 'selenium.webdriver.firefox.webdriver.WebDriver'
 
+    @classmethod
+    def setUpTestData(cls):
+        User.objects.create(
+            pk=100, username='super', first_name='Super', last_name='User', email='super@example.com',
+            password='sha1$995a3$6011485ea3834267d719b4c801409b8b1ddd0158', is_active=True, is_superuser=True,
```

Comment:
would be nice to comment what the plaintext version of this password is if it's not already